### PR TITLE
Update to add executable  file that would be installed into the PATH

### PR DESCRIPTION
**Pull request by gouniLee**
* Update to add executable `resbundler` file that would install into the PATH
* dependent package should be dependencies not `devDependencies`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +30,7 @@
     "url": "https://github.com/iLib-js/ilib-resbundler/issues"
   },
   "homepage": "https://github.com/iLib-js/ilib-resbundler#readme",
-  "devDependencies": {
+  "dependencies": {
     "ilib-common": "^1.0.1",
     "uglify-js": "^3.14.5"
```
Comment (ehoogerbeets) [inline package.json:37]: ilib-common is a dependency, but uglify-js is a devDependency

File: README.md [hunk 2]
```diff
@@ -32,4 +32,7 @@ resbundler.js --assembly dynamic --compiled compiled --resDir resources --outDir
 Release Notes
 -------------
 ### 1.0.0
-Intial release
\ No newline at end of file
+Intial release
+
+### 1.0.1
```
Comment (ehoogerbeets) [inline README.md:34]: The custom I have been following in the other projects it to put the most recent release notes at the top.

File: README.md [hunk 3]
```diff
@@ -32,4 +32,7 @@ resbundler.js --assembly dynamic --compiled compiled --resDir resources --outDir
 Release Notes
 -------------
 ### 1.0.0
-Intial release
\ No newline at end of file
+Intial release
```
Comment (ehoogerbeets) [inline README.md]: `Initial`

File: README.md (hunk 2 above)
Comment (gouniLee) [reply README.md:34]: I applied all of your comments

**Review by ehoogerbeets — APPROVED** [review]

**Pull request merged**